specs/olm.spec

```diff
@@ -17,6 +17,7 @@ Steps:
   * Validate triggers deployment
   * Verify TektonAddons Install status
   * Validate RBAC
+  * Assert if cronjob with default schedule and with default prefix present in target namespace
```

Comment:
We usually start steps with `Validate` or `Verify`, I like those two more than `Assert if`

Reply:
`Validate creation of default cronjob in target namespace`?? WDYT?